src/jdk.compiler/share/classes/com/sun/tools/javac/parser/DocCommentParser.java

```diff
@@ -1525,6 +1525,24 @@ private List<DCTree> tagAttrs() {
                 }
             },
 
+            // @spec url label
+            new TagParser(TagParser.Kind.BLOCK, DCTree.Kind.SPEC) {
+                @Override
+                public DCTree parse(int pos) throws ParseException {
+                    skipWhitespace();
+                    DCText url = inlineWord();
```

Comment:
Nit: call it `uri` for consistency.

Reply:
Update, the terminology is converging on `URL` in all public places, with `java.net.URI` being used as needed within the doclet implementation.